lib/gtest_1.7.0/src/gtest_main.cc

```diff
@@ -31,7 +31,18 @@
 
 #include "gtest/gtest.h"
 
+#include <stan/math/prim/arr/functor/mpi_cluster.hpp>
```

Comment:
Should this go under an `#ifdef STAN_MPI` as well?

Reply:
Oh, I see - that file is all within its own ifdef

Reply:
Correct. All MPI files are really safe to use in this regard.